src/coredump/coredump.c

```diff
@@ -1587,6 +1587,14 @@ static int process_kernel(int argc, char* argv[]) {
                 /* OK, now we know it's not the journal, hence we can make use of it now. */
                 log_set_target_and_open(LOG_TARGET_JOURNAL_OR_KMSG);
 
+        /* Log minimal metadata now, so it is not lost if the system is about to shut down. */
+        safe_atoi(context.meta[META_ARGV_SIGNAL], &signo);
+        log_info("PID %s (%s) crashed with signal %s/%s, processing coredump.",
```

Comment:
So this is still a bit too similar (and also too different) to me to the message we write when processing is done, which is:

```
        core_message = strjoin(
                        "Process ", context->meta[META_ARGV_PID],
                        " (", context->meta[META_COMM],
                        ") of user ", context->meta[META_ARGV_UID],
                        written ? " dumped core." : " terminated abnormally without generating a coredump.");
```

Also, I don#t like the word "crashed", because it's not clear what that means.

Please make this somewhat symmetric and use similar wording.

maybe something like this:

start: `"Process xyz (abc) of user def terminated abnormally, processing…"`
done: `"Process xyz (abc) of user def dumped core."`